draft-ietf-quic-recovery.md

```diff
@@ -348,14 +348,22 @@ latency at the peer or loss of previous ACK frames.  Any delays beyond the
 peer's max_ack_delay are therefore considered effectively part of path delay and
 incorporated into the smoothed_rtt estimate.
 
+The first ACK frame in the Handshake and ApplicationData packet number spaces
+can be greatly delayed due to a lack of decryption keys at the time they're
+received. For these two ACK frames, the max_ack_delay MAY be ignored if the
```

Comment:
It's not really the first, but any packets that are buffered, right?

Reply:
Sure, though hopefully you're processing all buffered packets before sending an ACK.